csv/csv.go

```diff
@@ -0,0 +1,99 @@
+package csv
+
+import (
+	"bytes"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/gobuffalo/pop"
+)
+
+// ErrInvalidCSV is returned when the input CSV file is not valid.
+var ErrInvalidCSV = errors.New("invalid CSV file")
+
+// Importer defines the options for the pop CSV import feature.
+type Importer struct {
+	cnx *pop.Connection
+	// Comma is the field delimiter.
+	// It is set to comma (',') by default.
+	// Comma must be a valid rune and must not be \r, \n,
+	// or the Unicode replacement character (0xFFFD).
+	Comma rune
+}
+
+// Import loads a CSV file from its name, and insert rows into the datasource
+// using the given connection.
+//
+// Example usage:
+// 	imp := csv.NewImporter(tx)
+// 	err := imp.Import("./csv/files/my_table_data.csv", "my_table")
+func (ci *Importer) Import(f string, model interface{}) error {
```

Comment:
@stanislas-m would it be better if instead of passing a local file path as param, this method can receive a `io.Reader`? This will allow not only import csv files locally but those located remotely (e.g some csv files downloaded with `net/http` package)